orientation_readme.md

```diff
@@ -0,0 +1,39 @@
+# Orientation : Welcome to your first day!
+This document was made to help newcomers set up all the software that they will need to actively learn and participate in the JustAskDavidB Coaching Program.
+
+##  Downloading and Setting Up Software
+
+### [Jetbrain](https://account.jetbrains.com/login)
+We use Jetbrain products for web development.  This includes PHPStorm for PHP development, DataGrip for SQL and database interaction, and Pycharm for Python development. Downloading the trials is fine because a JetBrain license will be purchased for you as part of the JustAskDavidB Coaching Program. You will receive an email from JetBrains to complete your registration within the first 15-days. After logging into JetBrains, “Authorization Successful” should appear on screen.
+
+#### 1. [PHPStorm](https://www.jetbrains.com/phpstorm/download/#section=windows)
+Download the program for your OS and run through the installation. After installing Pycharm, open Pycharm and run through the setup wizard. Do not worry there are no wrong answers :)
```

Comment:
"Phycharm" is used instead of "PHPStorm"
The last sentence is cute, but I'm not sure it really fits here. I recommend deleting, maybe substitute with something along the lines of "The installation should be relatively simple." (No smiley face)